src/org/exist/launcher/Launcher.java

```diff
@@ -97,17 +92,15 @@ public static void main(final String[] args) {
         SwingUtilities.invokeLater(() -> new Launcher(args));
     }
 
+    private ServiceManager serviceManager;
     private SystemTray tray = null;
     private TrayIcon trayIcon = null;
-    private Optional<WrappedService> runningAsService;
     private SplashScreen splash;
     private Optional<JettyStart> jetty = Optional.empty();
```

Comment:
The only write to the Optional may happen during initialization while the splash screen is shown. Additionally, the called methods of JettyStart itself are synchronized and conflicting menu items for start/stop are greyed out.